site/assets/scss/_boosted.scss

```diff
@@ -87,3 +87,246 @@ body {
     }
   }
 }
+
+// Accessibility statement
+
+.pie {
+  position: relative;
+  width: 200px;
+  height: 200px;
+  font-size: 3em;
+  text-align: center;
+  background: var(--bs-secondary-bg);
+  background-image: linear-gradient(to right, transparent 50%, var(--bs-primary) 0);
+  border-radius: 50%; // stylelint-disable-line property-disallowed-list
+}
+
+.pie::after,
+.pie::before {
+  position: absolute;
+  display: block;
+  width: 50%;
+  height: 100%;
+  margin-left: 50%;
+  content: "";
+  background-color: inherit;
+  border-radius: 0 100% 100% 0 / 50%; // stylelint-disable-line property-disallowed-list
+  transform-origin: left;
+}
+
+.pie::after {
+  background-color: var(--bs-primary);
+}
+
+.pie-val {
+  position: absolute;
+  top: 15px;
+  right: 15px;
+  bottom: 15px;
+  left: 15px;
+  z-index: 2;
+  padding-top: 25%;
+  background-color: var(--bs-body-bg);
+  border-radius: 50%; // stylelint-disable-line property-disallowed-list
+}
+
+.pie[data-value="1"]::before { transform: rotate(3.6deg); }
```

Comment:
When we'll have to modify the values, it might be a problem to find back the values isn't it? Except if we comment the useless ones and that the comments are stripped out, but I don't remember if that's the case.

Reply:
OK, it works with "//" comments (not multiline)